android/fastlane/metadata/android/fr-CA/changelogs/default.txt

```diff
@@ -0,0 +1 @@
+Améliore l'expérience utilisateur
```

Comment:
I've got two questions on this:
1- Is this what you were talking about that will be updated automatically from the release message in github?
2- Do this file has the correct encoding or it doesn't need to have it as it will be parsed back by the store?

Reply:
Very good question! 😄 

1. That is the default message if the release doesn't contain a message, you can check it is the same in English and for iOS. In  [this step](https://github.com/ApplETS/Notre-Dame/blob/feature/auto_deploy/.github/workflows/release-workflow.yaml#L75) of the `release-workflow.yaml` the CI will search for the right changelog to use and copy it in the correct file 😄
2. What do you mean by correct encoding? According to the Fastlane documentation, a simple string is sent to the store if that answers your question 😅 

I started (but not finished) writing the documentation in the wiki, you can see this [page](https://github.com/ApplETS/Notre-Dame/wiki/Github-Actions-CI-CD).

Reply:
1- Okayyy I see nice! 😄 
2- Oh, It seems like it was a problem on my part, Github was showing accents in "Améliore" as "Am@$liore" or something like that, don't know why but now since I came back it's fine!